src/pybind/mgr/dashboard/ci/check_grafana_dashboards.py

```diff
@@ -103,8 +103,6 @@ def get_grafana_dashboards(base_dir):
                 title = dashboard_config['title']
                 assert len(title) > 0, \
                     "Title not found in '{}'".format(json_file)
-                assert len(dashboard_config.get('links', [])) == 0, \
-                    "Links found in '{}'".format(json_file)
```

Comment:
What's the reason behind this change? Why do we need to remove this assert?

Reply:
@Pegonzal It is checking whether the links field in the Grafana dashboard configuration is empty or not,  and if it does, the script will raise an error, flagging that the json_file contains links. We tried creating an exception for the files in which the button was added, but it was still giving error in make check, so me and @aaSharma14 just decided to drop that assert as it is not needed.

Also i just added the button for multi-cluster and ceph-cluster dashboards in this PR since it was my first PR and i was just figuring out how it works. I will be adding it to the rest of the dashboards and will create a separate PR for the same.

Reply:
> just decided to drop that assert as it is not needed.

If this was not really needed I was trying to verify why was this here in the first place,
and traced it back to [this PR](https://github.com/ceph/ceph/pull/31951) by @epuertat, but could not find an explanation for the changes in that PR. Which, actually is just removing exactly what we are doing currently in this PR.
So unless @epuertat has any reason for it, this is okay with me as I don't have any.

> Also i just added the button for multi-cluster and ceph-cluster dashboards in this PR since it was my first PR and i was just figuring out how it works. I will be adding it to the rest of the dashboards and will create a separate PR for the same.

Sure, that's fine with me, ideally you could create a tracker so we don't miss on doing this, and if you already know you are going to work on it, just assign it to yourself and move it forward.

Reply:
Yeah i checked that PR by @epuertat as well. Do we really need that assert or can we go ahead with removing it? @epuertat 

And I'll create the tracker @Pegonzal